Source/WebKit/Platform/cocoa/CocoaHelpers.h

```diff
@@ -108,6 +108,7 @@ NSString *encodeJSONString(id, JSONOptionSet = { }, NSError ** = nullptr);
 NSData *encodeJSONData(id, JSONOptionSet = { }, NSError ** = nullptr);
 
 NSDictionary *dictionaryWithLowercaseKeys(NSDictionary *);
+NSDictionary *keepObjectsWithKeys(NSDictionary *, NSArray *keys);
```

Comment:
open to a better name for this

Reply:
`dictionaryContainingObjectsWithKeys()`?

Reply:
`dictionaryWithKeys()`?

Reply:
both are great options! leaning towards `dictionaryWithKeys`